tests/unit-tests/python/foglamp_test/services/core/api/test_scheduler.py

```diff
@@ -33,7 +33,7 @@
     await conn.execute(''' DELETE from foglamp.schedules WHERE process_name IN ('testsleep30', 'echo_test')''')
     await conn.execute(''' DELETE from foglamp.scheduled_processes WHERE name IN ('testsleep30', 'echo_test')''')
     await conn.execute('''insert into foglamp.scheduled_processes(name, script)
-        values('testsleep30', '["python3", "../scripts/sleep.py", "30"]')''')
+        values('testsleep30', '["python3", "../tests/unit-tests/python/foglamp_test/common/sleep.py", "30"]')''')
```

Comment:
kindly move this to contest as shared var $FOGLAMP_ROOT/tests/unit-tests/python/foglamp_test/common/ so that once tests are put at right place we can fix it at 1 place only. 

also do the same for core_managment_port.  @Singhal-Vaibhav  please feel free to own this PR and add moe commits.